tests/App/Entity/Media.php

```diff
@@ -19,6 +19,9 @@
 
 #[ORM\Entity]
 #[ORM\Table(name: 'media__media')]
+#[ORM\AttributeOverrides([
+    new ORM\AttributeOverride(name: 'length', column: new ORM\Column(type: Types::DECIMAL, precision: 2, nullable: true)),
```

Comment:
fails with PHP 8.0 :vomiting_face: 

Can we just update it here?

https://github.com/sonata-project/SonataMediaBundle/blob/e0436ccf5153e260a70d224930dd18da42682478/src/Resources/config/doctrine/BaseMedia.orm.xml#L17

Not sure how this mapping is used :man_shrugging:

Reply:
Adding `precision` is required with ORM 3 for decimal columns

Reply:
Will be fixed by https://github.com/sonata-project/SonataMediaBundle/pull/2448

Reply:
done https://github.com/sonata-project/SonataMediaBundle/releases/tag/4.13.0